src/Serilog.Sinks.EventLog/Sinks/EventLog/ICategoryNumberProvider.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2016 Serilog Contributors
```

Comment:
Nit - 2025 or just ©️

Reply:
Updated to 2025 on all new files in this PR